images-list

```diff
@@ -45,6 +48,8 @@ grafana/grafana-image-renderer rancher/grafana-grafana-image-renderer 2.0.0
 gcr.io/cloud-provider-vsphere/csi/release/driver rancher/mirrored-cloud-provider-vsphere-csi-release-driver v2.1.0
 gcr.io/cloud-provider-vsphere/cpi/release/manager rancher/mirrored-cloud-provider-vsphere-cpi-release-manager v1.2.1
 gcr.io/cloud-provider-vsphere/csi/release/syncer rancher/mirrored-cloud-provider-vsphere-csi-release-syncer v2.1.0
+gcr.io/google_containers/pause rancher/mirrored-pause 3.1
+gcr.io/google_containers/pause rancher/mirrored-pause 3.2
```

Comment:
Why aren't we including the org here ? `rancher/mirrored-google-containers-pause`

Reply:
I'm just adding the mirrored- prefix,it was pause prior.